js/ui/text_box/text_box.js

```diff
@@ -38,6 +38,14 @@ var TextBox = TextEditor.inherit({
             mode: "text",
 
             /**
+            * @name dxTextBoxOptions_valueChangeEvent
+            * @publicName valueChangeEvent
+            * @type string
+            * @default "change focusout"
+            */
+            valueChangeEvent: "change focusout",
```

Comment:
I think you can change default value for  "dxTextEditorOptions_valueChangeEvent" instead of adding new option here